flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/ExecutionVertex.java

```diff
@@ -736,7 +740,7 @@ void sendPartitionInfos() {
 	 * @return whether the input constraint is satisfied
 	 */
 	public boolean checkInputDependencyConstraints() {
```

Comment:
Can be package private.

Reply:
Changed it to default scope as it will be invoked by tests.